packages/eslint-plugin/rules/acronym-case.test.js

```diff
@@ -110,9 +110,6 @@ export const HTML = () => {};
 				{
 					message: '`useInstanceId` violates naming rules.',
 				},
-				{
-					message: '`useInstanceId` violates naming rules.',
```

Comment:
The invocation of `useInstanceId()` no longer results in an error while the declaration still does. Hence removing the second error message for the affected tests.